rooms-quickstart-java/src/main/java/com/communication/rooms/quickstart/App.java

```diff
@@ -72,128 +70,118 @@ public static boolean deleteRoom(String roomId)
-                .setRole(RoleType.CONSUMER);
-            ParticipantsCollection updatedParticipants = roomsClient.removeParticipants(roomId, List.of(firstChangeParticipant));
-            System.out.println("Participants: \n" + listParticipantsAsString(updatedParticipants.getParticipants()));
+            List<CommunicationIdentifier> participantsToRemove = new ArrayList<>();
+
+            participantsToRemove.add(new CommunicationUserIdentifier(USER_ID_2));
+
+            RemoveParticipantsResult removeParticipantsResult = roomsClient.removeParticipants(roomId,
+                    participantsToRemove);
+
+            System.out.println("Participant(s) removed");
         } catch (Exception ex) {
             System.out.println(ex);
         }
     }
 
-    public static void listParticipants(String roomId)
-    {
+    public static void listParticipants(String roomId) {
         try {
-            ParticipantsCollection participants = roomsClient.getParticipants(roomId);
-            System.out.println("Participants: \n" + listParticipantsAsString(participants.getParticipants()));
+            PagedIterable<RoomParticipant> participants = roomsClient.listParticipants(roomId);
+
+            for (RoomParticipant participant : participants) {
+                System.out.println(
+                        participant.getCommunicationIdentifier().getRawId() + " (" + participant.getRole() + ")");
+            }
+
         } catch (Exception ex) {
             System.out.println(ex);
         }
     }
 
-    private static String listParticipantsAsString(List<RoomParticipant> participants)
-    {
-        return "[" + participants.stream().map
-        (p -> "ID: " + p.getCommunicationIdentifier().getRawId()
-                    + "\nRole: " + p.getRole()
-        ).collect(Collectors.joining("\n")) + "]";
-    }
-
-
-    public static void main(String[] args)
-    {
+    public static void main(String[] args) {
         roomsClient = createRoomsClientWithConnectionString();
         int selection;
         Set<String> roomIds = new HashSet<>();
 
-        try
-        {
+        try {
             BufferedReader br = new BufferedReader(new InputStreamReader(System.in, "UTF-8"));
-        while(true)
-        {
+            while (true) {
 
                 System.out.println("Make a selection");
                 System.out.println("1. Add a room");
                 System.out.println("2. Update a room");
                 System.out.println("3. Delete a room");
                 System.out.println("4. Get room details");
-                System.out.println("5. List room ids");
-                System.out.println("6. Add a participant");
-                System.out.println("7. Update a participant");
+                System.out.println("5. List room ids created in this session");
+                System.out.println("6. Add or Update participants");
+                System.out.println("7. List rooms in resource");
```

Comment:
Can we swap 6 and 7? so we can have Rooms and Participants management separated in two groups.

Reply:
Swaped